src/seals/base_envs.py

```diff
@@ -148,13 +193,25 @@ def obs_from_state(self, state: State) -> State:
         return state
 
 
-class TabularModelMDP(ResettableMDP[int, int]):
-    """Base class for tabular environments with known dynamics."""
+# TODO(juan) this does not implement the .render() method,
```

Comment:
What I meant is that there was places in the seals/imitation source code where this object was used directly (I think). But now it feels absurd because that should clearly fail.

Reply:
Okay no, the methods that are instantiated directly and that don't implement render in imitation are `RandomMDP`, `CliffWorld`. Actually no environment in the imitation codebase implements render, but all environments are instantiated properly. Which is weird because Gym has it as an abstract method in its source code.